drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableConstants.properties

```diff
@@ -202,3 +202,5 @@ ResolvedHitPolicy=Resolved Hit
 ResolvedHitPolicyDescription=Similar to First Hit, but you can for example give row 10 priority over row 5. This means you can keep the order of the rows you want for visual readability, but specify priority exceptions.
 None=None
 HasPriorityOverRow=Has priority over row
+Convert=Convert
```

Comment:
Shouldn't we say `Convert to XLS`?

Reply:
We can. XLS tables had "Convert" so used it.